src/components/manage/Blocks/Search/components/SearchDetails.jsx

```diff
@@ -17,7 +17,11 @@ const SearchDetails = ({ total, text, as = 'h4' }) => {
   const intl = useIntl();
   return (
     <El className="search-details">
-      {text ? `${intl.formatMessage(messages.searchedFor)}: ${text}. ` : ''}
+      {text && (
+        <>
+          {intl.formatMessage(messages.searchedFor)}: <em>{text}</em>.{' '}
```

Comment:
I think `text` position should not be hardcoded. Some language could translate it as "*blabla* was searched". Check the syntax for intl.formatMessage.

Reply:
@tiberiuichim Yes, I agree. It's fixed.